share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -236,7 +238,7 @@ default:
     max: 2
     when:
       - always
-  tags: ["spack", "public", "medium", "x86_64"]
+  tags: ["spack", "public", "medium", "x86_64", "norun"]
```

Comment:
I'm not sure what this `norun` tag is, should it still be here?

Reply:
Title of the commit that added this was "Disable non-windows stacks using bad tags" LOL. Just going to drop this.